benchmark/opperf/utils/benchmark_utils.py

```diff
@@ -143,19 +145,35 @@ def run_performance_test(ops, inputs, run_backward=True,
 
 
 def run_op_benchmarks(ops, dtype, ctx, profiler, warmup, runs):
+    # Running SoftmaxOutput backwards on GPU results in errors
+    # track issue here: https://github.com/apache/incubator-mxnet/issues/880
+    gpu_backwards_disabled_ops = ['SoftmaxOutput']
+
+    # Running im2col either forwards or backwards on GPU results in errors
+    # track issue here: https://github.com/apache/incubator-mxnet/issues/17493
+    gpu_disabled_ops = ['im2col']
+
     # For each operator, run benchmarks
     mx_op_benchmark_results = []
     for op, op_params in ops.items():
-        # Prepare inputs for the operator
-        inputs = prepare_op_inputs(op, op_params)
-        # Run benchmarks
-        cur_op_res = run_performance_test(op_params["nd_op_handle"],
-                                          run_backward=op_params["has_backward"],
-                                          dtype=dtype, ctx=ctx,
-                                          profiler=profiler,
-                                          inputs=inputs,
-                                          warmup=warmup, runs=runs)
-        mx_op_benchmark_results += cur_op_res
+        if not (ctx == mx.gpu() and op in gpu_disabled_ops):
```

Comment:
Can we change the logic here to `ctx == mx.cpu() or op not in gpu_disabled_ops` for readability?